RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelDigisClustersFromSoA.cc

```diff
@@ -114,8 +114,7 @@ void SiPixelDigisClustersFromSoA::produce(edm::StreamID, edm::Event& iEvent, con
         edm::LogWarning("SiPixelDigisClustersFromSoA") << "cluster below charge Threshold "
                                                        << "Layer/DetId/clusId " << layer << '/' << detId << '/' << ic
                                                        << " size/charge " << acluster.isize << '/' << acluster.charge;
-      SiPixelCluster cluster(acluster.isize, acluster.adc, acluster.x, acluster.y, acluster.xmin, acluster.ymin);
-      cluster.setOriginalId(ic);
```

Comment:
remove ```setOriginalId``` from the interface of ```SiPixelCluster```?

Reply:
what about https://github.com/cms-patatrack/cmssw/blob/6a192beda960630de730923b10a44a5a1da5e77d/RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelClusterProducer.cc#L130 ?

Reply:
oops. sorry for the noise.
and cannot be done differently there...